src/Mapbender/CoreBundle/Resources/public/mapbender.element.legend.js

```diff
@@ -351,7 +361,7 @@
                 if(layers[layidx].children.length > sublayidx) {
                     if(layers[layidx].children[sublayidx].legend) {
                         $(self.element).find("#imgtest").html(
-                                '<img id="testload" style="display: none;" src="' + layers[layidx].children[sublayidx].legend.url + '"></img>');
+                            '<img id="testload" style="display: none;" src="' + layers[layidx].children[sublayidx].legend.url + '"></img>');
```

Comment:
Not sure if this belongs here, but:

While working on the legend-element anyways another little problem could be fixed at the same time:

Bigger Legend-Images are being cutted off on the right in Mapbender 3.0.6.3 (see screenshot enclosed)

I added this little Css to mapbender.element.gislegend.scss to fix it for GisInfoService with a scrollbar:

```
.mb-element-legend ul li.image {

  overflow: scroll;
![2017-07-20-legende_layertree](https://user-images.githubusercontent.com/24896163/29554883-15da4d18-8721-11e7-8b85-9f2247dbd4a3.png)

}
```